cmd/utils/flags.go

```diff
@@ -329,6 +330,21 @@ var (
 		Usage: "HTTP-RPC server listening port for the engineAPI",
 		Value: nodecfg.DefaultEngineHTTPPort,
 	}
+	EngineReadTimeoutFlag = cli.DurationFlag{
+		Name:  "engine.timeouts.read",
+		Usage: "Maximum duration for reading the entire request, including the body.",
+		Value: 30 * time.Second,
```

Comment:
`rpccfg.DefaultHTTPTimeouts` -- why not use these values here?

Reply:
I think it will cause import cycling with 'util' package in the future